### PR TITLE
Add vidcloud to flixhq and zoechip

### PR DESCRIPTION
**Pull request by JorrinKievit**
This pull request resolves #XXX

- Fixes duplicate Upcloud embed on zoechip.
- VidCloud has a CORS restriction so can't be used without the extension.

 - [x] I have read and agreed to the [code of conduct](https://github.com/movie-web/movie-web/blob/dev/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/movie-web/movie-web/blob/dev/.github/CONTRIBUTING.md).
 - [ ] What I'm implementing was assigned to me and is an [approved issue](https://github.com/movie-web/movie-web/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved). For reference, please take a look at our [GitHub projects](https://github.com/movie-web/movie-web/projects).
 - [x] I have tested all of my changes.

### CODE REVIEW

File: src/providers/sources/zoechip/common.ts [hunk 1]
```diff
@@ -55,6 +56,11 @@ export async function createZoeChipStreamData(ctx: MovieScrapeContext | ShowScra
   for (const source of sources) {
     const formatted = await formatSource(ctx, source);
     if (formatted) {
+      // Zoechip does not return titles for their sources, so we can not check if a source is upcloud or vidcloud because the domain is the same.
+      const upCloudAlreadyExists = embeds.find((e) => e.embedId === upcloudScraper.id);
+      if (formatted.embedId === upcloudScraper.id && upCloudAlreadyExists) {
+        formatted.embedId = vidCloudScraper.id;
+      }
```
Comment (mrjvs) [inline src/providers/sources/zoechip/common.ts:63]: how reliable is this?

Comment (JorrinKievit) [reply src/providers/sources/zoechip/common.ts:63]: I clicked on like 30 different shows and they all seem to have Upcloud & Vidcloud as servers. So I would say its reliable enough 🤔

**Review by JipFr — COMMENTED** [review]
I've tried ZoeChip for Dune and Dream Scenario, both of which returned UpCloud embeds that 404'ed when you actually click on them. Has it broken since you made the PR, or am I trying the wrong movies?

**Comment by JorrinKievit** [comment]
> I've tried ZoeChip for Dune and Dream Scenario, both of which returned UpCloud embeds that 404'ed when you actually click on them. Has it broken since you made the PR, or am I trying the wrong movies?

Hmm i did test dune and it was working before, ill check

**Comment by JorrinKievit** [comment]
> I've tried ZoeChip for Dune and Dream Scenario, both of which returned UpCloud embeds that 404'ed when you actually click on them. Has it broken since you made the PR, or am I trying the wrong movies?

Tested it and it seems to work for me here? 🤔 . Also tested in browser CLI which works (after adding CORS_ALLOWED flag temporarily)

image
image

**Pull request merged**